lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -488,28 +494,28 @@ def populate_api_routes( webapp, app ):
                            action='change_password',
                            conditions=dict( method=["GET"] ) )
 
-    webapp.mapper.connect( 'set_default_permissions',
-                           '/api/user_preferences/set_default_permissions',
+    webapp.mapper.connect( 'change_permissions',
+                           '/api/user_preferences/change_permissions',
                            controller='user_preferences',
-                           action='set_default_permissions',
+                           action='change_permissions',
                            conditions=dict( method=["GET"] ) )
 
-    webapp.mapper.connect( 'api_keys',
-                           '/api/user_preferences/api_keys',
+    webapp.mapper.connect( 'change_api_key',
+                           '/api/user_preferences/change_api_key',
                            controller='user_preferences',
-                           action='api_keys',
+                           action='change_api_key',
```

Comment:
I also don't get ``change_api_key`` shouldn't it be a ``PUT`` on ``api_keys``? Some of these endpoints seemed more RESTful before this change.